web-admin/src/routes/-/auth/device/+page.svelte

```diff
@@ -83,32 +85,44 @@
 </svelte:head>
 
 {#if user}
-  <div class="flex flex-col justify-center items-center h-3/5">
-    <h1 class="text-3xl font-medium text-gray-800 mb-4">
-      Hello, {user.displayName}!
-    </h1>
-    <p class="text-lg text-gray-700 mb-6">Your user code is: {userCode}</p>
-
-    <Button
-      type="primary"
-      on:click={() => {
-        actionTaken = true;
-        confirmUserCode();
-      }}
-      disabled={actionTaken}>Confirm</Button
-    >
-    <div class="mt-4" />
-    <Button
-      type="secondary"
-      on:click={() => {
-        actionTaken = true;
-        rejectUserCode();
-      }}
-      disabled={actionTaken}>Reject</Button
+  <div class="flex flex-col justify-center items-center h-4/5 gap-y-6">
```

Comment:
I bet this container will need to be reused in the future for the github connect / failed connection page.